pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -281,7 +281,12 @@ self: super: ({
 
   # https://github.com/fpco/unliftio/issues/87
   unliftio = dontCheck super.unliftio;
-
+  # same as above, but called from rio; 'xit' skips a test
+  rio = overrideCabal (drv: {
+    preConfigure = ''
+      sed -i 's/\bit /xit /g' test/RIO/FileSpec.hs
+    '';
+  }) super.rio;
```

Comment:
Could you report this upstream and put a link to the github issue here so we have some way of tracking when this is fixed?

Reply:
@cdepillabout - It's the same issue as above (https://github.com/fpco/unliftio/issues/87); do you want me to leave the same comment here?

Reply:
I'm not super familiar with the development of `rio` and `unliftio`, but it looks like they are developed in separate repos?

My guess is that posting an additional issue to the `rio` repo, so that we can track when the tests are fixed, would be helpful for us.

Although maybe you're trying to say that these failing `rio` tests would also be fixed as soon at the `unliftio` issue is fixed?  Like, this isn't an issue directly with `rio`, but it is just an effect of `unliftio` being broken?  In which case, could you update the issue so that the maintainers here in Nixpkgs would know that this override for `rio` could be removed as soon as the linked `unliftio` issue has been closed?

Reply:
@cdepillabout - Yes, sorry, should have been more clear. These failing `rio` tests will be fixed as soon as the `unliftio` tests are fixed; the issue is with `unliftio`, and `rio` just calls into the functions with an issue. I updated the comment here; is that better?

Reply:
Great, very easy to understand now!